### PR TITLE
add description section to response

### PR DESCRIPTION
**Pull request by X1ting**

### CODE REVIEW

File: app/views/apipie/apipies/_method_detail.erb [hunk 1]
```diff
@@ -40,6 +40,8 @@
   <%= heading(t('apipie.returns'), h_level) %>
   <% method[:returns].each do |item| %>
     <%= heading("#{t('apipie.code')}: #{item[:code]}", h_level + 1) %>
+    <%= heading("#{t('apipie.description')}:", h_level + 2) %>
+    <p><%= item[:description] %></p>
```
Comment (iNecas) [inline app/views/apipie/apipies/_method_detail.erb]: Shouldn't we check on the presence of the `item[:description]` before adding this?

Comment (iNecas) [reply app/views/apipie/apipies/_method_detail.erb]: @X1ting ping?

Comment (X1ting) [reply app/views/apipie/apipies/_method_detail.erb]: Ough, sorry, forgot to answer. Will change tomorrow. Thanks for review!

**Comment by iNecas** [comment]
Thanks @X1ting, test failures unrelated

**Comment by iNecas** [comment]
apipie-rails-0.5.14 has just been released, including this fix

**Pull request merged**